packages/pnpm/src/main.ts

```diff
@@ -202,7 +202,7 @@ export default async function run (inputArgv: string[]) {
       // Don't check for updates
```

Comment:
Should this clarifying comment be updated because the if-statement now has more checks?